.github/workflows/update-bullet-train.yml

```
@@ -0,0 +1,83 @@
+# This workflow will prepare a PR that pulls in the latest changes from the
+# Bullet Train starter repo.
+name: "Create Bullet Train Update PR"
+on:
+  workflow_call:
+  workflow_dispatch:
+    inputs:
+      versionNumber:
+        description: 'Version Number'
+        required: true
+        type: string
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

There's a newer version of this package, might as well 😄

```suggestion
        uses: actions/checkout@v4
```